docs/content/docs/guide/repositorycrd.md

```diff
@@ -205,3 +205,156 @@ and a pull request event.
 - [Github Documentation for webhook events](https://docs.github.com/webhooks-and-events/webhooks/webhook-events-and-payloads?actionType=auto_merge_disabled#pull_request)
 - [Gitlab Documentation for webhook events](https://docs.gitlab.com/ee/user/project/integrations/webhook_events.html)
 {{< /hint >}}
+
+## Scope Github token to a list of private and public repositories within and outside namespaces
+
+By default, the Github token that Pipelines as Code generates is scoped only to the repository where the payload comes from.
+However, in some cases, the developer team might want the token to allow control over additional repositories.
+For example, there might be a CI repository where the `.tekton/pr.yaml` file and source payload might be located, however the build process defined in `pr.yaml` might fetch tasks from a separate private CD repository.
+
+You can extend the scope of the Github token in two ways:
+
+- _Repository level configuration_: extend the Github token to a list of repositories that exist in the same namespace as the original repository.
+
+- _Global configuration_: extend the Github token to a list of repositories in different namespaces.
```

Comment:
I think you may want to add that it is a admin configuration setting and the other is user setting (and reverse the order maybe starting with global and go after with repo level)

Reply:
Updated